src/Cli/Commands/ConfigureOptions.cs

```diff
@@ -19,13 +19,43 @@ namespace Cli.Commands
     public class ConfigureOptions : Options
     {
         public ConfigureOptions(
+            string? dataSourceDatabaseType = null,
+            string? dataSourceConnectionString = null,
+            string? dataSourceOptionsDatabase = null,
+            string? dataSourceOptionsContainer = null,
+            string? dataSourceOptionsSchema = null,
+            bool? dataSourceOptionsSetSessionContext = null,
             int? depthLimit = null,
             string? config = null)
             : base(config)
         {
+            DataSourceDatabaseType = dataSourceDatabaseType;
+            DataSourceConnectionString = dataSourceConnectionString;
+            DataSourceOptionsDatabase = dataSourceOptionsDatabase;
+            DataSourceOptionsContainer = dataSourceOptionsContainer;
+            DataSourceOptionsSchema = dataSourceOptionsSchema;
+            DataSourceOptionsSetSessionContext = dataSourceOptionsSetSessionContext;
             DepthLimit = depthLimit;
         }
 
+        [Option("data-source.database-type", Required = false, HelpText = "Database type. Allowed values: MSSQL, PostgreSQL, CosmosDB_NoSQL, MySQL.")]
+        public string? DataSourceDatabaseType { get; }
+
+        [Option("data-source.connection-string", Required = false, HelpText = "Connection string for the data source.")]
+        public string? DataSourceConnectionString { get; }
+
+        [Option("data-source.options.database", Required = false, HelpText = "Database name for Cosmos DB for NoSql.")]
+        public string? DataSourceOptionsDatabase { get; }
+
+        [Option("data-source.options.container", Required = false, HelpText = "Container name for Cosmos DB for NoSql.")]
+        public string? DataSourceOptionsContainer { get; }
+
+        [Option("data-source.options.schema", Required = false, HelpText = "Schema path for Cosmos DB for NoSql.")]
+        public string? DataSourceOptionsSchema { get; }
```

Comment:
@sajeetharan, can you provide feedback on this help text for Cosmos? if i recall, we need to specify Azure Cosmos DB instead of Cosmos DB for NoSql?

Reply:
Should be Azure Cosmos DB for NOSQL (full caps)

Reply:
why full caps for NoSQL? for example:
![image](https://github.com/user-attachments/assets/3c51bb51-a445-4264-bf2e-1a4ddb45cc63)
![image](https://github.com/user-attachments/assets/1afd6470-2fff-4eb1-a33a-4d7e55a3f1e4)

Also i feel fullcaps is not that readable.